src/workloads/workloads.csproj

```diff
@@ -1,9 +1,9 @@
-﻿<Project DefaultTargets="Restore;Build">
+<Project DefaultTargets="Restore;Build">
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
 
   <PropertyGroup>
     <MicrosoftDotNetBuildTasksInstallersTaskTargetFramework Condition="'$(MSBuildRuntimeType)' == 'Core'">net7.0</MicrosoftDotNetBuildTasksInstallersTaskTargetFramework>
-    <MicrosoftDotNetBuildTasksInstallersTaskTargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net5.0</MicrosoftDotNetBuildTasksInstallersTaskTargetFramework>
+    <MicrosoftDotNetBuildTasksInstallersTaskTargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net6.0</MicrosoftDotNetBuildTasksInstallersTaskTargetFramework>
```

Comment:
@am11 Could you please check this?

Reply:
This is for source-build, which is why I left it out from the patch. I think it can be updated later, when source-build requires it.

The end-to-end test for source build runs in dotnet/installer repo; first the  PRs are merged in dotnet/runtime repo, dotnet/sdk is updated with latest runtime, and finally installer repo is updated with SDK where source-build happens in the CI. Therefore, it is not immediately obvious whether it is OK to make this change.

Reply:
Thanks for the information,  I couldn't have done it without you 😍
Ok, I'll revert this.

Reply:
The Installers package uses net7.0 in both the normal build and the source build: https://github.com/dotnet/arcade/blob/8e8cab10677494713f64937650d4996782223740/src/Microsoft.DotNet.Build.Tasks.Installers/build/Microsoft.DotNet.Build.Tasks.Installers.props#L3. Ideally we would not hardcode the tfm here at all, but we can do that in a separate change if we would like to.

Reply:
Thank Viktor @ViktorHofer I changed it to 7.0